docs/documentation/pages_en/quickstart.md

```diff
@@ -265,15 +267,22 @@ Or use one of the following instructions to set up the local Kubernetes cluster
     
    **IMPORTANT** Param `--insecure-registry` allows usage of Container Registry without TLS. TLS in our case dropped for simplicity.
 
-3. Install NGINX Ingress Controller:
+3. If you haven't installed `kubectl` yet, install it by running:
+
+   ```
+   minikube kubectl -- get po -A
```

Comment:
Why do we need this command before creating an alias?

Reply:
This command pumps out kubectl and installs it in the system. It is given in the official installation [instructions](https://minikube.sigs.k8s.io/docs/start/). With the same condition: if minicube is not installed yet, then...

Reply:
This doc says:
> Alternatively, minikube can download the appropriate version of kubectl and you should be able to use it like this:
> minikube kubectl -- get po -A …

I don't feel it means you actually need to execute this command before creating the alias.